Library/Homebrew/language/node.rb

```diff
@@ -32,13 +40,13 @@ def self.setup_npm_environment
       end
     end
 
-    def self.std_npm_install_args(libexec)
+    def self.std_npm_install_args(libexec, prepublishRequiresDeps=false)
```

Comment:
Should this stay an explicit opt-in or do we want to do this unconditionally? Note that later would add a for most formula unnecessary overhead of installing all dependencies twice (before pack and during install).

Also any suggestions for a better variable name?